docs/content/release-notes/flink-1.15.md

```diff
@@ -0,0 +1,379 @@
+propagated into `TableConfig`. Even though configuration set directly in `TableConfig` 
+has still precedence, this change can have side effects if table configuration 
+was accidentally set in other layers.
+
+### [Remove pre FLIP-84 methods (FLINK-26090)](https://issues.apache.org/jira/browse/FLINK-26090)
+
+The previously deprecated methods `TableEnvironment.execute`, `Table.insertInto`,
+`TableEnvironment.fromTableSource`, `TableEnvironment.sqlUpdate`, and 
+`TableEnvironment.explain` have been removed. Please use the provided 
+alternatives introduced in 
+[FLIP-84](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=134745878).
+
+### [Fix parser generator warnings (FLINK-26053)](https://issues.apache.org/jira/browse/FLINK-26053)
+
+`STATEMENT` is a reserved keyword now. Use backticks to escape tables, fields and 
+other references.
+
+### [Expose uid generator for DataStream/Transformation providers (FLINK-25990)](https://issues.apache.org/jira/browse/FLINK-25990)
+
+`DataStreamScanProvider` and `DataStreamSinkProvider` for table connectors received 
+an additional method that might break implementations that used lambdas before. 
+We recommend static classes as a replacement and future robustness.
+
+### [Add new STATEMENT SET syntax (FLINK-25392)](https://issues.apache.org/jira/browse/FLINK-25392)
+
+It is recommended to update statement sets to the new SQL syntax: 
+
+```SQL
+EXECUTE STATEMENT SET BEGIN ... END;
+EXPLAIN STATEMENT SET BEGIN ... END;
+```
+
+### [Check & possible fix decimal precision and scale for all Aggregate functions (FLINK-24809)](https://issues.apache.org/jira/browse/FLINK-functions)
+
+This changes the result of a decimal `SUM()` with retraction and `AVG()` between
+1.14.0 and 1.14.1. It restores the behavior of 1.13 to be consistent with Hive/Spark.
+
+### [Clarify semantics of DecodingFormat and its data type (FLINK-24776)](https://issues.apache.org/jira/browse/FLINK-24776)
+
+The `DecodingFormat` interface was used for both projectable and non-projectable
+formats which led to inconsistent implementations. The `FileSystemTableSource` 
+has been updated to distinguish between those two interfaces now. Users that 
+implement custom formats for `FileSystemTableSource` might need to verify the 
+implementation and make sure to implement `ProjectableDecodingFormat` if necessary.
+
+### [Update user document for batch window tvf (FLINK-24760)](https://issues.apache.org/jira/browse/FLINK-24760)
+
+All window table-valued functions can now also be used in batch mode.
+
+### [Add user document for Window Deduplication (FLINK-24656)](https://issues.apache.org/jira/browse/FLINK-24656)
+
+Flink SQL now also supports Window Deduplication as a new window table-valued function.
+`WindowDeduplication` is a deduplication operator which removes rows that duplicate
+over a set of columns, keeping only the first one or the last one for each window
+and other partitioned keys.
```

Comment:
I don't think these 2 belong in here. They are ultimately new features, and not a cause for issues during upgrades.